config/deploy.rb

```diff
@@ -51,4 +51,20 @@
   end
   before :restart, :resquepoolrestart
 
+  # load the workflow configs
+  desc "Load workflow configurations"
+  task :loadworkflows do
+    on roles(:web) do
+      execute :rake, 'curation_concerns:workflow:load'
+  end
+  before :restart, :loadworkflows
+
```

Comment:
what the heck is this 'workflow configuration' stuff, new in new version of Sufia/CC? Doc'd anywhere you know of?